applications/Anchor.md

```diff
@@ -0,0 +1,286 @@
+    //  T::BlockNumber, last block number when updated data successfully.
+    pub(super) type AnchorOwner<T: Config> = StorageMap<_, Twox64Concat, Vec<u8>, (T::AccountId,T::BlockNumber)>;
+
+    // (T::AccountId, u32,T::AccountId)
+    // T::AccountId, the anchor owner ss58 address
+    // u32, the sell price for the anchor
+    // T::AccountId, the target buyer. If the same as owner, it is free to buy.
+    pub(super) type SellList<T: Config> = StorageMap<_, Twox64Concat, Vec<u8>, (T::AccountId, u32,T::AccountId)>;
+  ```
+
+### anchor.js
+
+- exposed methods. Three parts : basic substrate functions, anchor data I/O functions, anchor exchange functions. It has been used in the demo freeSaying, but no isolated yet. And exchange part need to code this time.
+Anchor.js can help developer accessing Anchor network by this single JS.
+
+  ```JS
+    /*******************************/
+    /* substrate related functions */
+    /*******************************/
+
+    //link to target substrate node
+    exports.link=function(endpoint,callback){
+      /* return callback && callback(PolkadotJS.API) */
+    };
+
+    //load encry file to get pair
+    exports.load=function(file,password,callback){
+      /* return callback && callback(pair) */
+    };
+
+    //subcribe the latest block
+    exports.subcribe=function(callback){
+      /* return callback && callback(anchorDataList) */
+    };
+
+    //check account balance
+    exports.balance=function(ss58Address,callback){
+      /* return callback && callback(amount) */
+    },
+    
+    /***********************************/
+    /* Anchor pallet related functions */
+    /***********************************/
+
+    /* data part */
+    //get the latest anchor data
+    exports.latest=function(anchor,callback){
+      /* return callback && callback(formattedAnchorData) */
+    };
+
+    //get the anchor data on special block.
+    exports.target=function(anchor,block,callback){
+      /* return callback && callback(formattedAnchorData) */
+    };
+
+    //get the list data of anchor.
+    exports.history=function(anchor,callback,limit){
+      /* return callback && callback(listofAnchorsData) */
+    };
+
+    //set target Anchor data.
+    exports.write=function(pair,anchor,raw,protocol,callback){
+      /* return callback && callback(toChainProcessStatus) */
+    };
+
+    /* market part */
+    //set Anchor status to sell.
+    exports.sell=function(pair,anchor,cost,target,callback){
+      /* return callback && callback(true/false) */
+    };
+
+    //buy anchor on sell.
+    exports.buy=function(pair,anchor,callback){
+      /* return callback && callback(true/false) */
+    };
+
+    //revoke anchor from selling.
+    exports.unsell=function(pair,anchor,callback){
+      /* return callback && callback(true/false) */
+    };
+  ```
+
+### Overview
+
+- **Total Estimated Duration:** 1 month
+- **Full-Time Equivalent (FTE):**  1
+- **Total Costs:** 6,000USD
```

Comment:
```suggestion
- **Total Costs:** 6,000 USD
```

Reply:
Hello, @keeganquigley , sorry for the slow response.
I will modify it and update.